tests/abort2_case1.rs

```diff
@@ -149,13 +149,8 @@ async fn test_abort_case_2_move_on_with_other_makers() {
 
     // Maker might not get banned as Taker may not try 6102 for swap. If it does then check its 6102.
     if !taker.read().unwrap().get_bad_makers().is_empty() {
-        let onion_addr_path = PathBuf::from(format!("/tmp/tor-rust-maker{}/hs-dir/hostname", 6102));
-        let mut file = File::open(onion_addr_path).unwrap();
-        let mut onion_addr: String = String::new();
-        file.read_to_string(&mut onion_addr).unwrap();
-        onion_addr.pop();
```

Comment:
Let's keep this code alive and put it under the if clause if taker.config.connectiontype = tor. In that way we can later change the test to tor if required. Same for other tests.

Reply:
The sole reason I considered removing this part was because some of the guys from the sob channel were experiencing issues with the tor setup. My idea was to make both libtor and mitosis optional dependencies, depending on the feature integration.

Reply:
even in that case, we can use this code later to test with Tor if we want to. Removing it seems too hard and removes our ability to selectively test tor in future.